project/Game.py

```diff
@@ -71,9 +82,111 @@ def test_completed_game(self):
         self.game.end()
         self.assertTrue(self.game.ended, "Game Has Ended")
 
+class landmarkDummy:
+    clue = "The Place we drink coffee and read books"
+    question = "What is the name of the statue out front?"
+    answer = "three disks"
+    location = "uwm library"
+    points = 150
+    timer = datetime.time(0,0,15)
+    timepenelty = 20
+    answerpenalty = 10
+
+class teamDummy:
+    points = 100
+    currentLandmark = 1
+    timelog = [datetime.time(0,20,15),datetime.time(0,35,25)]
+    password = "password"
+    clueTime = datetime.time(0,0,0)
+
+class Test_Game_Team(unittest.TestCase):
+    def setUp(self):
+        self.game = Game()
+        l1 = landmarkDummy()
+        l2 = landmarkDummy()
+        l3 = landmarkDummy()
+        self.game.landmarks=[l1,l2,l3]
+        self.game.curteam = teamDummy()
+        self.game.penaltyTime = 20
+        self.game.penaltyValue = 10
+        self.game.timer = datetime.time(00,00,15)
+
+    def test_get_status(self):
+        self.game.curteam.clueTime = datetime.time(5,30,50)    
```

Comment:
I think you're using curteam to avoid having to deal with the list. The problem is who updates this and when? What if there is no curteam (gamemaker is logged in and wants all statuses)?